stdlib/3/types.pyi

```diff
@@ -57,16 +62,12 @@ class CodeType:
             cellvars: Tuple[str, ...] = ...,
     ) -> None: ...
 
-class MappingProxyType:
-    def copy(self) -> dict: ...
-    def get(self, key: str, default: _T = ...) -> Union[Any, _T]: ...
-    def items(self) -> Iterator[Tuple[str, Any]]: ...
-    def keys(self) -> Iterator[str]: ...
-    def values(self) -> Iterator[Any]: ...
-    def __contains__(self, key: str) -> bool: ...
-    def __getitem__(self, key: str) -> Any: ...
-    def __iter__(self) -> Iterator[str]: ...
+class MappingProxyType(Mapping[_KT, _VT], Generic[_KT, _VT]):
+    def __init__(self, mapping: Mapping[_KT, _VT]) -> None: ...
+    def __getitem__(self, k: _KT) -> _VT: ...
```

Comment:
I'm surprised you have to define `__getitem__` and `__iter__` here, since they are defined in the base class you use (`Mapping`).